examples/graphbolt/rgcn/download.py

```diff
@@ -0,0 +1,608 @@
+import argparse, hashlib, os, shutil, tarfile, yaml
+import subprocess
+import urllib.request as ur
+
+import dgl.graphbolt as gb
+import numpy as np
+from tqdm import tqdm
+
+GBFACTOR = float(1 << 30)
+
+
+def build_yaml_helper(path, dataset_size, in_memory=True):
+    """The stirng to build the yaml file. (Still need modification)"""
+    data = {
+        "graph": {
+            "nodes": [
+                {"num": num_nodes[dataset_size]["paper"], "type": "paper"},
+                {"num": num_nodes[dataset_size]["author"], "type": "author"},
+                {
+                    "num": num_nodes[dataset_size]["institute"],
+                    "type": "institution",
+                },
+                {
+                    "num": num_nodes[dataset_size]["fos"],
+                    "type": "field_of_study",
+                },
+            ],
+            "edges": [
+                {
+                    "path": "edges/author__affiliated_to__institute.npy",
+                    "type": "author:affiliated_to:institution",
+                    "format": "numpy",
+                },
+                {
+                    "path": "edges/paper__written_by__author.npy",
+                    "type": "paper:written_by:author",
+                    "format": "numpy",
+                },
+                {
+                    "path": "edges/paper__cites__paper.npy",
+                    "type": "paper:cites:paper",
+                    "format": "numpy",
+                },
+                {
+                    "path": "edges/paper__topic__fos.npy",
+                    "type": "paper:has_topic:field_of_study",
+                    "format": "numpy",
+                },
+            ],
+        },
+        "tasks": [
+            {
```

Comment:
Can we also add the 2nd 2K task? One can have the name `node_classification_19`, the other can have the name `node_classification_2983`. Or if you have an idea on how to better name these 2 alternatives, you can go ahead.

Reply:
Sure. I will add another task on 2983 classes

Reply:
1 task 1 run. In the training script, we can choose the task[1] too if we want to.

Reply:
But it is also technically possible to do 1 run 2 tasks at the same time.

Reply:
Got it. Do we need to do 1 run 2 tasks or it is fine to keep it 1 task 1 run for now. Currently we should support to run the 2983 task asa long as we change tasks[0] to tasks[1]

Reply:
1 run 1 task is more than sufficient in the example so long as the dataset contains both tasks.